### ISSUE TITLE
Naming inconsistencies across Data.(List/Vec).All.Properties

### ISSUE THREAD

**Issue by gallais** [agda/agda-stdlib#306]
Compare [`Data.List.*`'s prefix-free `++⁺`](https://agda.github.io/agda-stdlib/Data.List.All.Properties.html#6368) to [`Data.Vec.*`'s prefixed `All-++⁺`](https://agda.github.io/agda-stdlib/Data.Vec.All.Properties.html#2003).

**Comment by MatthewDaggitt** [comment]
Yup... it was inconsistent when I started, and as I've been going along I've been adding things prefix-free, as that seemed to the standard in `List.All` and `List.Any` which is where most of the existing proofs of this form were.

I'm not particularly wedded to either form. We should be consistent though. From a time perspective, I think going prefix-free would be easier?

**Comment by gallais** [comment]
I would also pick the prefix-free version. Importing a module qualified
is the best way for a user to introduce prefixes in case disambiguation
is needed.
